Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Core.cpp

```diff
@@ -277,6 +278,17 @@ namespace ScriptCanvasEditor
         return m_path;
     }
 
+    void SourceHandle::SanitizePaths()
+    {
+        m_path.MakePreferred();
+        m_relativePath = m_path.RelativePath();
+    }
+
+    const AZ::IO::Path& SourceHandle::RelativePath() const
```

Comment:
where does this function get used?

Reply:
So far, no where. I just added it. But it will be made use of in MainWindow and in the Reflected Property Editor a lot.